resources/themes/cura-light/icons/low/Check.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M10.7 13.3L7.4 10L6 11.4L10.7 16.1L17.4 9.4L16 8L10.7 13.3Z" fill="#000E1A"/>
```

Comment:
The `clip-rule` property is not supported by Qt.
The `fill-rule` property is not used in this image since there are no self-intersections.
The `fill` property is not used in this image, since it is always recoloured. The colour shown here is incorrect (different from what we actually use in the interface).